PokeTraining/models.py

```diff
@@ -91,12 +111,27 @@ class Pokemon(models.Model):
         ('FAIRY', 'Fairy')
     )
 
+    ALL_LANGUAGES = (
```

Comment:
Add support for other supported languages (i.e. Japanese, Italian, French, Spanish, Chinese, etc.)

Reply:
#5 will be addressing this change.